lib/src/BaseTools/ClockTicks.H

```diff
@@ -59,10 +59,14 @@ inline unsigned long long int ch_ticks()
 }
 #define CH_TICKS
 #else
-// if we fall all the way through to here, we don't really do anything,
-// but we still need the NamespaceHeader to balance
-// the namespacefooter. (DFM 4/28/09)
+#include <time.h>
 #include "BaseNamespaceHeader.H"
+inline unsigned long long int ch_ticks()
+{
+  timespec temp;
+  clock_gettime(CLOCK_MONOTONIC, &temp);
+  return temp.tv_sec * 1000000000l + temp.tv_nsec;
+}
```

Comment:
Even though this environment variable isn't used much, we should probably still define it.

```suggestion
}
#define CH_TICKS
```